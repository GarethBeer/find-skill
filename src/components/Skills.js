import React from "react";
import Map from "./Map";

const Skills = () => {
  return (
    <div>
      <Map />
    </div>
  );
};

export default Skills;
