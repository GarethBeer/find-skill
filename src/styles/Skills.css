.skills{
    display: flex;
    flex-direction: row;
    justify-content: space-evenly;
}

.map{
    width: 70vw;
    height: 89vh;
    order: 1;
}

.sidebar {
    width: 30vw;
    order: 2;
    display: flex;
    flex-direction: column;
    flex-wrap: wrap;
    justify-content: space-evenly;   
}
.skillcard-container {
    border: solid 1px rgb(5, 107, 5);
    align-content: center;
    min-height: 36vh;
    flex-shrink: 0;
}
.infotext {
    text-align: center;
    color: lawngreen;
    text-shadow: 1px 2px black ;
    font-size: 25px;
    padding: 3vw;
}
@media only screen and (max-width: 732px) {
    .skills{
       flex-direction: column;
    }
    .map{
        width: 98%;
        order: 2;
    }
    .sidebar {
     width: 100%;
     order:1;
     flex-direction: row;
    }
    .skillcard-container {
        order: 4;
        width: 99%;
    }
}