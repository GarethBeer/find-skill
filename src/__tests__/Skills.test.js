import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter as Router } from "react-router-dom";
import Skills from "../components/Skills";

describe("Skills", () => {
  it("renders correctly to match the Skills snapshot", () => {
    const { asFragment } = render(
      <Router>
        <Skills />
      </Router>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
