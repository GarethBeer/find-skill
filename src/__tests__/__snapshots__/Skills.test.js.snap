// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skills renders correctly to match the Skills snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="map"
    >
      <div
        style="width: 100%; height: 100%; margin: 0px; padding: 0px; position: relative;"
      >
        <div
          style="width: 100%; height: 100%; left: 0px; top: 0px; margin: 0px; padding: 0px; position: absolute;"
        />
        <div
          style="width: 50%; height: 50%; left: 50%; top: 50%; margin: 0px; padding: 0px; position: absolute;"
        >
          <div
            style="width: 100%; height: 100%; left: 0px; top: 0px; margin: 0px; padding: 0px; position: absolute;"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
